### ISSUE TITLE
geordi commit: Pass arbitrary arguments to the git command

### ISSUE THREAD

**Issue by makmic** [makandra/geordi#71]
It would be nice if geordi would relay every unused argument for `geordi commit` to the underlying git command. This can be helpful i.e. to create an initially empty commit with the sole purpose to be squashed upon:

```bash
git checkout -b my-new-branch
geordi commit
# nothing to commit, working directory clean
```

If geordi would relay my arguments to git, the following would be possible:

```bash
git checkout -b my-new-branch
geordi commit --alow-empty
# [my-new-branch] Pivotal Story Message here
```

**Comment by denzelem** [comment]
Thanks @makmic for your suggestion. I think it would make sense to allow this in Geordi.

As you can see there are some more issues open, which partially have a higher priority. That means it could take a while until this feature will be implemented.

Nevertheless I will be happy to review a PR in case you have time to implement this on your own.

**Comment by makmic** [comment]
Thank you for looking into and priorizing the issue. I do not expect to free up time in the near future to create a PR by myself, but I'll let you know if i do :wink:

**Comment by codener** [comment]
@makmic In the just released 2.7.0, `geordi commit` will make empty commits by default (but print a warning). Arguments are forwarded to `git` nontheless and allow further customization.

**Comment by makmic** [comment]
Thank you for looking into this! :-)
